Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,60 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "without-test", "Skip running build-time tests (not recommended)"
+  depends_on "cmake" => :run
+  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
```

Comment:
sure thing.

The intention of my grouping was to make it clear that I was over-riding the message associated with the `--without-robodoc` install option to be less cryptic and that the effect of passing that option would be that the API documentation would not get built and would not be installed.